docs/requirements.txt

```diff
@@ -13,11 +17,3 @@ scipy
 setproctitle
 tensorflow
 tensorflow-probability
-
-# dev dependencies
-matplotlib
-recommonmark
-sphinx
-sphinx-autoapi>=1.4.0
-sphinx_rtd_theme
-sphinxcontrib-bibtex
```

Comment:
These deleted dependencies are installed by readthedocs implicitly and for now this is the only way I can get pip to install everything correctly